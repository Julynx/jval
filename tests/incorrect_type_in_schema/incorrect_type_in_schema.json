{
    "role": "user",
    "name": "Alice",
    "cars": [
        {
            "type": "car",
            "brand": "Toyota",
            "model": "Corolla",
            "year": "two thousand twenty"
        }
    ]
}