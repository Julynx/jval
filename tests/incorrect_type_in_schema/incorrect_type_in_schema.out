Validation error at 'cars[0].year': expected type <int> for key 'year'